setupPads.js

```diff
@@ -1,39 +1,50 @@
-import {drumPads, context, fileInputs} from "./constants.js";
-import {uploadFile} from "./uploadSamples.js";
+import { drumPads, context, fileInputs } from "./constants.js";
+import { uploadFile } from "./uploadSamples.js";
 let source;
 
 export function assignSoundsToPads(bufferList) {
   console.log("aSTP bufferList", bufferList);
 
   for (let i = 0; i < bufferList.length; i++) {
-    console.log('assigning', bufferList[i].name + " to pad " + i);
+    console.log("assigning", bufferList[i].name + " to pad " + i);
 
     makeLabel(drumPads[i], bufferList[i].name);
 
     let makeSourceFromBufferAndPlay = (e) => {
       //stop propagation to prevent sample from playing twice on mobile
-      e.stopPropagation();e.preventDefault();
+      e.stopPropagation();
+      e.preventDefault();
       let bufferSource = makeSource(bufferList[i]);
       console.log("playing", bufferSource);
       bufferSource.source.start(0);
     };
 
     drumPads[i].addEventListener("mousedown", makeSourceFromBufferAndPlay);
     drumPads[i].addEventListener("touchstart", makeSourceFromBufferAndPlay);
-              //chage to i, after adding all buttons
-    fileInputs[i].addEventListener("change", ()=>{
-      console.log("removed default sample from pad")
-      drumPads[i].removeEventListener("mousedown", makeSourceFromBufferAndPlay);
-      drumPads[i].removeEventListener("touchstart", makeSourceFromBufferAndPlay);
-    }, {once: true})
+    //chage to i, after adding all buttons
+    fileInputs[i].addEventListener(
+      "change",
+      () => {
+        console.log("removed default sample from pad");
+        drumPads[i].removeEventListener(
+          "mousedown",
+          makeSourceFromBufferAndPlay
+        );
+        drumPads[i].removeEventListener(
+          "touchstart",
+          makeSourceFromBufferAndPlay
+        );
+      },
+      { once: true }
+    );
   }
 }
 
-export function makeSource(buffer){
+export function makeSource(buffer) {
   source = context.createBufferSource();
   source.buffer = buffer;
   source.connect(context.destination);
-  return {source: source};
+  return { source: source };
```

Comment:
In JS when they variable is the same name as the key you can use this shorthand: 

` { source }` is equivalent to `{ source: source }`